javascript/javascript1/week3/nonoyes.js

```diff
@@ -0,0 +1,57 @@
+// Save a note
+
+const notes = [];
+function saveNote(content, id) {
+  notes.push({ content, id });
+}
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+// Get a note
+
+function getNote(id) {
+  for (let note of notes) {
+    if (note.id == isNaN(id)) {
+      return "Error";
+    } else {
+      for (let note of notes) {
+        if (note.id === id) return note;
+      }
```

Comment:
Why are you looping through the whole array again? you are already in the loop and have access to every element of the array